database.json

```diff
@@ -0,0 +1,6 @@
+{
+    "hostname": "TODO.postgres.database.azure.com",
+    "username": "TODO",
```

Comment:
how is this working? 
we shouldn't push to main while this is still TODO. 
is this json used? can't we retrieve this info in other ways?

Reply:
I dont know how we can use this file. Its similar to the config.json. Is it better to remove this file?